standalone-metastore/metastore-server/src/main/java/org/apache/hadoop/hive/metastore/ObjectStore.java

```diff
@@ -11286,6 +11287,20 @@ private String getPrimaryKeyConstraintName(String catName, String dbName, String
     return notNullConstraints;
   }
 
+  @Override
+  public SQLAllTableConstraints getAllTableConstraints(String catName, String db_name, String tbl_name)
+      throws MetaException {
+    debugLog("Get all table constraints for the table - " + catName + "."+ db_name+"."+tbl_name + " in class ObjectStore.java");
+    SQLAllTableConstraints sqlAllTableConstraints = new SQLAllTableConstraints();
```

Comment:
Let's have a followup jira to have overall one db call to fetch all constraints instead of having one for each type of constraint.

Reply:
https://issues.apache.org/jira/browse/HIVE-24062